wandb/sdk/artifacts/artifact.py

```diff
@@ -2102,6 +2140,48 @@ def _load_manifest(self, url: str) -> None:
                 dep_artifact = entry._get_referenced_artifact(self._client)
                 self._dependent_artifacts.add(dep_artifact)
 
+    @staticmethod
+    def _get_gql_artifact_fragment() -> str:
+        api = InternalApi()
+        fields = api.server_create_artifact_introspection()
+        fragment = """
+        fragment ArtifactFragment on Artifact {
+            id
+            artifactSequence {
+                project {
+                    entityName
+                    name
+                }
+                name
+            }
+            versionIndex
+            artifactType {
+                name
+            }
+            description
+            metadata
+            ttlDurationSeconds
+            aliases {
+                artifactCollection {
+                    project {
+                        entityName
+                        name
+                    }
+                    name
+                }
+                alias
+            }
+            state
+            commitHash
+            fileCount
+            createdAt
+            updatedAt
+        }
```

Comment:
Nit: increase the indentation by 2 spaces so that it easy to spot where the string begins and ends.